sys/compat/freebsd64/freebsd64_abort2.c

```diff
@@ -42,7 +44,8 @@ freebsd64_abort2(struct thread *td, struct freebsd64_abort2_args *uap)
 {
 	void * __capability uargs[16];
 	void *uargsp;
-	long ptr;
+	uint64_t * __capability uargscap;
```

Comment:
Well, it's single-use in the source, but within a for loop, and since `fueword_c` is a black box to this translation unit that could be messing with whatever the `uap` passed in points to, the whole expression would instead need to be recomputed every time. This is already a minor issue with the existing code since it does a pointless load on every iteration rather than keeping `uap->args` in a local, but adding in the capability (re)construction compounds that.